frontend/src/metabase/components/Button.styled.tsx

```diff
@@ -18,6 +17,6 @@ export const TextButton = styled(Button)<{ size: "small" | "medium" }>`
 
   &:hover {
     background-color: transparent;
-    color: ${color("text-brand")};
+    color: var(--mb-color-brand);
```

Comment:
`text-brand` does not exist today, so this hover logic has never resulted in a brand color before. Should we remove this line altogether to keep existing behaviour?